aidial_adapter_openai/gpt4_multi_modal/download.py

```diff
@@ -23,30 +24,55 @@
 
 def guess_attachment_type(attachment: dict) -> Optional[str]:
     type = attachment.get("type")
-    if type is None:
-        return None
 
-    if "octet-stream" in type:
+    if type is None or "octet-stream" in type:
         # It's an arbitrary binary file. Trying to guess the type from the URL.
         url = attachment.get("url")
         if url is not None:
             url_type = mimetypes.guess_type(url)[0]
             if url_type is not None:
                 return url_type
-        return None
```

Comment:
I think we arranged on one of previous PRs, that even if statement will still fall into returning None anyway, it's better to make this `return None` explicit

Reply:
I would have agreed if the method ended right here, however, it goes on up to the explicit return: `return type`.